usecases/node_api.py

```diff
@@ -41,3 +41,39 @@ def get_address_search(
         except HathorCoreTimeout:
             self.node_api_gateway.blacklist_address(address)
             return ADDRESS_BLACKLIST_RESPONSE
```

Comment:
Yes, the previous response was "This address is blacklisted" but the reason of the blacklist is a timeout due to too many transactions. This message was changed because it is showed to the user on the frontend so it should reflect the actual reason.

Reply:
> but the reason of the blacklist is a timeout due to too many transactions

I'm not sure I understand this sentence.

I brought an example from line 18 in this file to make my point clearer:
```
        if self.node_api_gateway.is_blacklisted_address(address):
            return ADDRESS_BLACKLIST_RESPONSE
```

I don't understand why in this case we return `Timeout due to too many transaction` to the user.